pimd/pim6_cmd.h

```diff
@@ -39,6 +39,8 @@
 #define IFACE_PIM_HELLO_TIME_STR "Time in seconds for Hello Interval\n"
 #define IFACE_PIM_HELLO_HOLD_STR "Time in seconds for Hold Interval\n"
 #define MROUTE_STR "IP multicast routing table\n"
+#define CLEAR_IP_IGMP_STR "IGMP clear commands\n"
```

Comment:
Do we need this in pim6_cmd.h? Please remove it.

Reply:
Resolved, Thanks